templates/win2k12r2-deprecated.tpl.yaml

```diff
@@ -58,6 +58,7 @@ metadata:
         }, {
           "name": "windows-cd-bus",
           "path": "jsonpath::.spec.domain.devices.disks[*].cdrom.bus",
+          "valid": "jsonpath::.spec.domain.devices.disks[*].cdrom.bus",
```

Comment:
"jsonpath::.spec.domain.devices.disks[*].cdrom" is better, otherwise you will miss when somebody adds cdrom, but forgets to set the bus.

Also double check that [*] works for valid parameter.

Reply:
fixed